ui_framework/.babelrc

```diff
@@ -1,3 +0,0 @@
-{
-  "presets": ["react", "@spalger/babel-presets"]
-}
```

Comment:
The removal of this file causes the following error when I run `uiFramework:start`:

![image](https://cloud.githubusercontent.com/assets/1238659/24517926/f13d62a0-1534-11e7-8e3d-9a773af28092.png)